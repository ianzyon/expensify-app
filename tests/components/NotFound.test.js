import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import NotFound from '../../src/components/NotFound';

test('should render NotFound page correctly', ()=>{
    // add component instance 
    const wrapper = shallow(<NotFound />);

    expect(toJSON(wrapper)).toMatchSnapshot();
});
