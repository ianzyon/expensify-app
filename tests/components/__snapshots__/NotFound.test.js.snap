// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound page correctly 1`] = `
<div>
  404: Page Not Found - 
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
